packages/kbn-esql-validation-autocomplete/src/autocomplete/recommended_queries/suggestions.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type { SuggestionRawDefinition, GetFieldsByTypeFn } from '../types';
+import { getRecommendedQueries } from './templates';
+
+export const getRecommendedQueriesSuggestions = async (
+  getFieldsByType: GetFieldsByTypeFn,
+  fromCommand: string = ''
+): Promise<SuggestionRawDefinition[]> => {
+  const fieldSuggestions = await getFieldsByType('date', [], {
+    openSuggestions: true,
+  });
+  let timeField = '';
+  if (fieldSuggestions.length) {
+    timeField =
+      fieldSuggestions?.find((field) => field.label === '@timestamp')?.label ||
+      fieldSuggestions[0].label;
+  }
+
+  const recommendedQueries = getRecommendedQueries({ fromCommand, timeField });
+
+  const suggestions: SuggestionRawDefinition[] = recommendedQueries.map((query) => {
+    return {
+      label: query.label,
+      text: query.queryString,
+      kind: 'Issue',
```

Comment:
I used the Issue because is the same we use in the date picker suggestion and the bucket. So I like the consistency part. Also have you seen this icon? Is very bad 

<img width="616" alt="image" src="https://github.com/user-attachments/assets/1ab8fc87-b1cb-41b6-b5a7-c4c669a35f1b">